test/controllers/CheckYourAppealController.test.ts

```diff
@@ -1,19 +1,20 @@
 import 'reflect-metadata';
 
-import { Arg } from '@fluffy-spoon/substitute'
-import { expect } from 'chai';
-import { INTERNAL_SERVER_ERROR, MOVED_TEMPORARILY, OK } from 'http-status-codes';
+import {Arg} from '@fluffy-spoon/substitute'
+import {expect} from 'chai';
+import {INTERNAL_SERVER_ERROR, MOVED_TEMPORARILY, OK} from 'http-status-codes';
 import * as request from 'supertest';
 
 import 'app/controllers/CheckYourAppealController';
-import { Appeal } from 'app/models/Appeal';
-import { Email } from 'app/modules/email-publisher/Email';
-import { EmailService } from 'app/modules/email-publisher/EmailService'
-import { CHECK_YOUR_APPEAL_PAGE_URI, CONFIRMATION_PAGE_URI } from 'app/utils/Paths';
+import {Appeal} from 'app/models/Appeal';
+import {Email} from 'app/modules/email-publisher/Email';
+import {EmailService} from 'app/modules/email-publisher/EmailService'
+import {AppealStorageService} from 'app/service/AppealStorageService';
+import {CHECK_YOUR_APPEAL_PAGE_URI, CONFIRMATION_PAGE_URI} from 'app/utils/Paths';
 
-import { createApp, getDefaultConfig } from 'test/ApplicationFactory';
-import { createSubstituteOf } from 'test/SubstituteFactory'
-import { createFakeSession } from 'test/utils/session/FakeSessionFactory';
+import {createApp, getDefaultConfig} from 'test/ApplicationFactory';
+import {createSubstituteOf} from 'test/SubstituteFactory'
+import {createFakeSession} from 'test/utils/session/FakeSessionFactory';
```

Comment:
You seem to have removed all the spaces from in between the imports :(

Reply:
must be my .editorconfig 🤔